app/src/app/[lng]/[inventory]/page.tsx

```diff
@@ -384,24 +392,25 @@ export default function Home({ params: { lng } }: { params: { lng: string } }) {
                     />
                     <Box>
                       <Box className="flex gap-1">
-                        {inventory?.city.area! == 0 || inventory?.city.area === null ? (
-                           <Text
-                           fontFamily="heading"
-                           color="border.neutral"
-                           fontSize="headline.sm"
-                           fontWeight="semibold"
-                           lineHeight="32"
-                         >
-                           N/A
-                         </Text>
+                        {inventory?.city.area! == 0 ||
+                        inventory?.city.area === null ? (
```

Comment:
It would make sense to do the null check first here, and use === for comparison with 0.